ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomInputField.js

```diff
@@ -27,13 +27,19 @@ const CustomInputField = (props) => {
       <TextField
         InputLabelProps={{ style : { pointerEvents : "auto", padding : "3px" } }}
         variant="outlined"
+        style={{ width : "15rem" }}
         size="small" autoFocus
         key={props.id}
         value={props.value}
         id={props.id}
         onChange={e => props?.onChange(e.target.value)}
         label={`${prettifiedName}`}
         InputProps={{ style : { padding : "2px 0px 0px 0px" },
+          inputProps : {
+            type : 'number',
+            max : 100,
+            min : 0
+          },
```

Comment:
This actually modifies all the input fields to accept numbers, and that too between 0-100, right @Chadha93, @sudo-NithishKarthik ?
Should it be desired? replicas may go over 100 too?

Reply:
yes, @Abhishek-kumar09 It's the desired behavior, we are capping the number of replicas to 100, I think no user will make more replicas than that  (it's a big range for replicas) : )

Reply:
Yeah the concern is we are changing all the inputFields, some fields may only want string and we are pushing to take numbers between 0-100. Though this problem would not have been there before liveValidate, But after @sudo-NithishKarthik new PR, this can be a problem accepting all input fields to contain numbers. 
Rather change in uiSchema that is a rjsf specific property is desired.

Reply:
@Abhishek-kumar09 yes totally makes sense, you are right it will have an issue with live validation, is issue 1(setting the range or defining the strict type as a number) valid to implement this way or we should not go with that?

Reply:
@Abhishek-kumar09 updated, this time it wont affect all the string fields only field required number with range will be changed